input/postgres/relation_stats.go

```diff
@@ -72,6 +73,18 @@ SELECT s.indexrelid,
 			 AND ($1 = '' OR (s.schemaname || '.' || s.relname) !~* $1)
 `
 
+const columnStatsSQL = `
+SELECT schemaname, tablename, attname, inherited, null_frac, avg_width, n_distinct, correlation
+  FROM %s
```

Comment:
I think we can ignore pg_catalog here, i.e.

```sql
SELECT schemaname, tablename, attname, inherited, null_frac, avg_width, n_distinct, correlation
  FROM %s
 WHERE schemaname <> 'pg_catalog'
```

Reply:
Good catch. Looks like `information_schema` should be skipped too.